components/cloud-event-gateway-proxy/cmd/main.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
its a best practice to move this code to a subfolder with the name of the application that you want to create:
e.g: cmd/cloud-event-gateway-proxy/main.go
otherwise a simple go build ./cmd will lead to a binary called cmd ...

Reply:
Done.